tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -416,4 +416,27 @@ describe(__filename, () => {
 
     expect(root.find('.AddonMoreInfo-version-history')).toHaveLength(0);
   });
+
+  it('renders the last updated date', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      last_updated: Date.now(),
+    });
+    const root = render({ addon });
+
+    expect(root.find('.AddonMoreInfo-last-updated')).toHaveLength(1);
+    expect(root.find('.AddonMoreInfo-last-updated').children()).toIncludeText(
+      'a few seconds ago',
```

Comment:
Would it be safer to check that it includes today's date, rather than the text `a few seconds ago`? I'm just wondering if there could possibly be timing issues that could produce an intermittent failure.

Reply:
I thought about it too but this component displays relative time here, so I am not sure I can use a fixed date.